app/helpers/podcast_planner_helper.rb

```diff
@@ -1,32 +1,20 @@
 # frozen_string_literal: true
 
 module PodcastPlannerHelper
-  PERIODIC_WEEKS = I18n.t([:every_one, :every_two, :every_three, :every_four], scope: [:podcast_planner, :helper, :period_options])
-  MONTHLY_WEEKS = I18n.t([:first, :second, :third, :fourth, :fifth], scope: [:podcast_planner, :helper, :monthly_options])
```

Comment:
Translating these in the helper wasn't working in staging for some reason?  They always showed up as translation-missing.  So converted to doing them just-in-time here.